WcaOnRails/app/webpacker/components/DelegateTab/RoleForm.jsx

```diff
@@ -89,10 +102,22 @@ function RoleForm({
         <Form.Button
           primary
           type="submit"
-          disabled={disabled || JSON.stringify(formValues) === JSON.stringify(savedValues)}
+          disabled={disabled
+            || saving
+            || JSON.stringify(formValues) === JSON.stringify(savedValues)}
```

Comment:
Would be good to extract this as a const since it's used twice, here and in the cancel button.

Reply:
Done, thank you.